utils/stack-templates/deploy.sh

```diff
@@ -5,16 +5,15 @@ set -e
 ################################################################################
 # repo
 ################################################################################
-helm repo add {{HELM_REPO_NAME}} {{HELM_REPO_URL}}
+helm repo add datawire https://getambassador.io
```

Comment:
I don't think you meant to modify this file :)

Reply:
Good catch ;) I was wondering why my changes didn't take effect the first time.